pkg/rpcserver/local.go

```diff
@@ -100,23 +98,6 @@ func RunLocal(cfg *LocalConfig) error {
 		cmd.Process.Kill()
 		<-res
 	}
-	if !cfg.HandleInterrupts {
```

Comment:
> It's assumed that the caller would use a context to control waits on
individual requests.

The runtest tests use context.Background() to wait on requests.

Reply:
Refactored runtest to accept a context.